acceptance-tests/src/test/resources/uk/gov/di/test/features/IPN.feature

```diff
@@ -107,9 +107,9 @@ Feature: International Phone Numbers
     Then the user is shown an error message
     When the user clicks the support link due to their security code not working
     Then the user is taken to the Contact us page in a new tab
-    When the user selects "A problem signing in to your GOV.UK One Login" and proceeds
+    When the user selects "A problem signing in to your GOV.UK One Login"
     Then the user is taken to the "A problem signing in to your GOV.UK One Login" page
-    When the user selects "The security code did not work" and proceeds
+    When the user selects "The security code did not work"
     Then the user is taken to the "The security code does not work" page
-    When the user selects "Text message to a phone number from another country" and proceeds
-    Then the user receives confirmation that their message has been submitted
\ No newline at end of file
+    And the user selects "Text message to a phone number from another country"
```

Comment:
Similar to above. Does this `And` offer us anything?

Reply:
Response same as above.